numba/tests/test_nrt.py

```diff
@@ -554,6 +554,10 @@ def foo(x):
 class TestNrtExternalCFFI(MemoryLeakMixin, TestCase):
     """Testing the use of externally compiled C code that use NRT
     """
+    def setUp(self):
+        # initialize the NRT (in case the tests are run in isolation)
+        super(TestNrtExternalCFFI, self).setUp()
+        cpu.CPUContext(typing.Context())
```

Comment:
We should not create a new `CPUContext`. Better to use 

```python
from numba.core.registry import cpu_target
cpu_target.target_context
```
to force the initialization of the `CPUContext`

Reply:
Good point, I forgot this was recently changed. There's another instance of this in this test module, I'll fix that too.